### PR TITLE
Updates for utils/fuse_appearance_embeddings_into_shs_dc

### PR DESCRIPTION
**Pull request by hardikdava**
@yzslab I have fixed few things in this script.

- used detach for saving plys
- updated import with latest code

### CODE REVIEW

**Comment by hardikdava** [comment]
@yzslab Right now, it is loading all the data to cpu/gpu. The script did not work on laptop gpu. I had to use bigger machine to work with large dataset. Can you suggest how can I make it lazy image loading for this script. I suggest as following.

- making lazy dataset loading
- make prunning optional

**Comment by yzslab** [comment]
> @yzslab Right now, it is loading all the data to cpu/gpu. The script did not work on laptop gpu. I had to use bigger machine to work with large dataset. Can you suggest how can I make it lazy image loading for this script. I suggest as following.
> 
> * making lazy dataset loading
> * make prunning optional

Hi, this script only load cameras into GPU, and it will not consume too much GPU memory.
Though pruning is optional, but calculating the visibilities is required by weighted fusing, so you must use a machine equipped with a NVIDIA GPU.


To reduce GPU memory consumption:
* Try reducing the [--camera-chunk-size](https://github.com/yzslab/gaussian-splatting-lightning/blob/845dcb421fef87e4c4c166f5e0a6527fdc4a746e/utils/fuse_appearance_embeddings_into_shs_dc.py#L22-L23) if you are using [wighted color mode](https://github.com/yzslab/gaussian-splatting-lightning/blob/845dcb421fef87e4c4c166f5e0a6527fdc4a746e/utils/fuse_appearance_embeddings_into_shs_dc.py#L24)
* Or switch to [weighted embedding mode](https://github.com/yzslab/gaussian-splatting-lightning/blob/845dcb421fef87e4c4c166f5e0a6527fdc4a746e/utils/fuse_appearance_embeddings_into_shs_dc.py#L271-L279)

**Comment by yzslab** [comment]
@hardikdava Hi, take a look the comment of `internal/utils/gaussian_utils.py`

**Comment by yzslab** [comment]
Hi, I have made a modification to your `internal/utils/gaussian_utils.py`. Check whether it works.

**Comment by yzslab** [comment]
> @hardikdava Hi, take a look the reveiw comment of `internal/utils/gaussian_utils.py` above.

Sorry, I found I forgot to submit the review yesterday.

File: internal/utils/gaussian_utils.py [hunk 1]
```diff
@@ -182,12 +182,12 @@ def to_ply_format(self):
         assert isinstance(self.xyz, torch.Tensor) is True
         return GaussianPlyUtils(
             sh_degrees=self.sh_degrees,
-            xyz=self.xyz.cpu().numpy(),
-            opacities=self.opacities.cpu().numpy(),
```
[the remark this answers is not in the document]
Comment (hardikdava) [reply internal/utils/gaussian_utils.py]: @yzslab yeah, I think `@torch_no_grad()` will fix `detach()` issue.

[the remark this answers is not in the document]
Comment (hardikdava) [reply internal/utils/gaussian_utils.py]: @yzslab `to_ply_format` function was called when I converted ckpt2ply from utils.

**Comment by hardikdava** [comment]
@yzslab sorry for late reply. Added my comments to your questions.

**Review by yzslab — APPROVED** [review]

**Comment by yzslab** [comment]
Thanks!

**Pull request merged**